docs/src/main/sphinx/release/release-444.md

```diff
@@ -0,0 +1,97 @@
+# Release 444 (2 Apr 2024)
+
+## General
+
+* Fix failure for queries containing large numbers of `LIKE` terms in boolean
+  expressions. ({issue}`21235`)
+
+## Docker image
+
+* Update Java runtime to Java 22. ({issue}`21161`)
+
+## BigQuery connector
+
+* Fix failure when reading BigQuery views with [Apache
+  Arrow](https://arrow.apache.org/docs/). ({issue}`21337`)
+
+## ClickHouse connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Delta Lake connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned ACLs with the native S3 file system. ({issue}`21176`)
+* Add concurrent write reconciliation for non-blind `INSERT`. ({issue}`20983`)
```

Comment:
I'm not sure I understand what this means...

Reply:
Me neither ..  any quick idea @martint  .. might have to read more about it

Reply:
No idea. @ebyhr, @findinpath? What's the user visible impact? Does it fix a failure, correctness or performance issue, or add a new feature? Under what scenarios?

Reply:
This is about allowing multiple writers to simultaneously perform operations which don't collide with one another.

A non-blind Delta Lake INSERT operations is an operation which selects from the same table as the target table.

e.g. `INSERT INTO t SELECT * FROM t where part = 10`


So to give a concrete example what the above mentioned PR does,
assuming that the table `t` is partition on the column `part`,
then the following operations can be successfully written in parallel 

```
INSERT INTO t SELECT * FROM t where part = 10
INSERT INTO t SELECT * FROM t where part = 20
INSERT INTO t SELECT * FROM t where part = 30
```

Before the above mentioned PR, this was not possible.

Reply:
What was the previous behavior? Query failure? Data corruption?

Reply:
In Delta Lake the transaction files are having the form:
```
1.log
2.log
3.log
...
```

Assuming that the a writer started reading the table at version 3, when committing the table it was attempting to create a new file `4.log` . In case another concurrent transaction won the race for `4.log`, the current operation until now would have failed.
With this PR, the connector attempts to "reconcile" to see whether the current transaction can go on unaffected by the changes from `4.log` and if it is so, it attempts to create a file `5.log`.

Reply:
Thanks. I would suggest rephrasing as "Add support for concurrent, non-conflicting writes when a table is read and written to in the same query".